tests/api/test_all_stories.py

```diff
@@ -228,6 +229,8 @@
             'meta': {
                 'story_related_nodes_backward': [0, 0, 0, 1, 0, 1],
                 'story_related_nodes_forward': [0, 1, 0, 0, 0, 0],
+                'wait_times': [[23, 16, 5], [-24, 7, 54], [0, 19, 8], [869, 22, 54],
+                               [-29726, 5, 56]],
```

Comment:
I wonder why there are negative values in the `wait_times`. Am I missing something?

Reply:
@yashvardhannanavati It is because the test dates are not accurate to what the real pipeline would look like. I did the math for each and they check out, but since the fake data is not in the real order it returns weird numbers like this.

Reply:
Is it because the wait_time is calculated using `start_time - completion_time`?

Reply:
Ah, makes sense. It would be great to fix a few just to be sure. :)

Reply:
@sarah256 would it make sense to check `wait_time < timedelta()`? If it is, then it's an invalid timedelta and an exception can be raised. Any code that calls that function would need to ignore it though.

Reply:
Also, to @yashvardhannanavati's point, could you please add a test or edit an existing one that has valid timestamps?